src/ds++/DracoMath.hh

```diff
@@ -6,13 +6,8 @@
  * \brief  New or overloaded cmath or cmath-like functions.
  * \note   Copyright (C) 2016 Los Alamos National Security, LLC.
  *         All rights reserved.
- *
- * History:
- * 2013-10-17 Many small header files were combined to create DracoMath.hh.
  */
 //---------------------------------------------------------------------------//
-// $Id$
```

Comment:
Can be done in git: https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes#Keyword-Expansion